src/manifolds/Grassmann.jl

```diff
@@ -145,11 +145,11 @@ b_{i}=\begin{cases}
 \end{cases}
 ````
 """
-function distance(M::Grassmann, p, q)
+function distance(::Grassmann, p, q)
     p ≈ q && return zero(real(eltype(p)))
     a = svd(p' * q).S
     a[a .> 1] .= 1
-    return sqrt(sum((acos.(a)) .^ 2))
+    return sqrt(sum(x -> abs2(acos(x)), a))
```

Comment:
Actually, minor point, but this would be even better than clamping in-place:
```suggestion
    return sqrt(sum(x -> abs2(acos(min(x, 1))), a))
```

Reply:
And then leave out the line 151? Sure. I could also use `clamp(x,-1,1)`?

Reply:
I'm pretty sure singular values are also always non-negative by construction, so the lower bound check should never be necessary, but I don't think it will make much of a performance difference.

Reply:
Then I'll stick with clamp for now.